library/src/main/java/org/xmtp/android/library/Conversation.kt

```diff
@@ -184,6 +202,7 @@ sealed class Conversation {
             return when (this) {
                 is V1 -> conversationV1.topic.description
                 is V2 -> conversationV2.topic
+                is Group -> TODO()
```

Comment:
Is the id() in V3 the same as a topic? Can we turn the byte array into a string?